test/openssl/test_pkey.rb

```diff
@@ -433,17 +433,6 @@ def test_s_from_parameters_dh_with_invalid_parameter
       e = assert_raise(OpenSSL::PKey::PKeyError) { OpenSSL::PKey.from_parameters("DH", invalid: 1234) }
       assert_match(/Invalid parameter "invalid"/, e.message)
     end
-
-    def test_s_from_parameters_ed25519
-      key = OpenSSL::PKey.from_parameters("ED25519", pub: "\xD0\x8E\xA8\x96\xB6Fbi{$k\xAC\xB8\xA2V\xF4n\xC3\xD06}R\x8A\xE6I\xA7r\xF6D{W\x84")
```

Comment:
IMO better to use `omit_on_fips` rather than deleting these tests entirely. There's precedent for it :) https://github.com/ruby/openssl/blob/a8caa63729e66f8ad5cb503f0199e099042faac5/test/openssl/test_pkey.rb#L84-L86

Reply:
I had two options and chose the delete option :)

I reasoned that the implementation is pretty generic and does not maybe make sense to test all the different algorithms via this new mechanism.

Happy to bring back that one but maybe the one testing the invalid param could be left out

Reply:
I now ended up with a compromise, so left one test with a non-FIPS algorithm.

Reply:
Thank you! I agree with your reasoning, but also happy to see this test back even if it's only to serve as documentation on how to use it in Ruby. I've learned a lot from the gem's test suite on how to do certain things, whereas https://www.openssl.org/docs/man3.0/man3/EVP_PKEY_fromdata.html is less straightforward to get a working example.

Reply:
I also can agree with the `test_s_from_parameters_ed25519` using `omit_on_fips`. This test can be used as a documentation for people to know ED25519 is not supported in FIPS.